run-make-check.sh

```diff
@@ -70,8 +70,17 @@ function main() {
         return 1
     fi
     FOR_MAKE_CHECK=1 prepare
+    cxx_compiler=g++
+    c_compiler=gcc
+    for i in $(seq 11 -1 10); do
```

Comment:
i s/11/14/ to be more future-proof. the latest version of clang is 12. see https://releases.llvm.org/download.html. and 13 is already packaged by some distros.